librad/src/paths.rs

```diff
@@ -40,9 +41,7 @@ impl Paths {
         .init()
     }
 
-    // Don't use system paths, but the supplied directory as a root.
-    //
-    // For testing, you know.
+    /// All paths are contained in the given directory.
     pub fn from_root(root: impl AsRef<Path>) -> Result<Self, io::Error> {
```

Comment:
In relation to https://github.com/radicle-dev/radicle-link/pull/496/files#r565931339, this doesn't take into account the `id`, so this https://github.com/radicle-dev/radicle-link/pull/496/files#diff-b5c21e30928026061fdc46f4a876c7bc8374ebe78fe68851375632dffe20d472R83 line seems off to me. You're creating the id, but not using it.

Reply:
You’re right, it does not take the profile ID into account. We could add the profile ID as a second argument and join it to root. But I’d rather prefer to do that at the call site, that is `Profile::from_root`. The latter isn’t implemented yet as @MeBrei  correctly pointed out.

Reply:
Fixed in f36bfc9a8aad7acdc637f5abd913a1c4dba61915